wrappers/java/build.gradle

```diff
@@ -6,6 +6,33 @@ version '1.0-SNAPSHOT'
 
 sourceCompatibility = 1.8
 
+// build extra start script for EraViewer
```

Comment:
This doesn't build and probably you need 

apply plugin: 'java'
apply plugin: 'application'

at the top of the file and to rename second buildEraViewerScript into buildEraNetworkConfig.

Reply:
"apply plugin" is deprecated syntax. Please go back to using "plugins {}"